attendance/views.py

```diff
@@ -42,7 +42,16 @@ def user_check(request, username):
         return True
     else:
         return False
-
+        
+def last_month(this_month=datetime.now().month):
+	if(this_month > 0 and this_month <= 12):
```

Comment:
You Can write it  more nicely in python :)  
```python
if 0 < this_month <= 12:
```
Also do not use brackets if it is not necessary (like after if statements, for loops and so on)